packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -641,11 +641,10 @@ class TreemapNode extends TreeNode<TreemapNode> {
     return TextSpan(
       text: '$displayName$separator[${prettyByteSize()}]',
       children: [
-        if (caption != null)
-          TextSpan(
-            text: '$separator$caption',
-            style: TextStyle(fontStyle: FontStyle.italic, color: textColor),
-          )
+        TextSpan(
```

Comment:
Why were these changes introduced?

Reply:
Oh whoops, it didn't update from the previous changes. Will revert!